pyproject.toml

```
@@ -29,7 +29,8 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-space_packet_parser = ">=4.0.2,<5"
+xarray = '>=0.0.0'
```

just remove the version dependency if you're going to put it at 0s?

or some more recent xarray is likely to be required.
```suggestion
xarray = '>=2023.0.0'
```